specification/security/put_role/SecurityPutRoleRequest.ts

```diff
@@ -38,12 +38,34 @@ export interface Request extends RequestBase {
     refresh?: Refresh
   }
   body?: {
+    /**
+     * A list of application privilege entries.
+     */
     applications?: ApplicationPrivileges[]
+    /**
+     * A list of cluster privileges. These privileges define the cluster-level actions for users with this role.
+     */
     cluster?: string[]
+    /**
+     * An object defining global privileges. A global privilege is a form of cluster privilege that is request-aware. Support for global privileges is currently limited to the management of application privileges.
+     */
     global?: Dictionary<string, UserDefinedValue>
+    /**
+     * A list of indices permissions entries.
+     */
     indices?: IndicesPrivileges[]
+    /**
+     * Optional metadata. Within the metadata object, keys that begin with an underscore (`_`) are reserved for system use.
+     */
     metadata?: Metadata
+    /**
+     * A list of users that the owners of this role can impersonate.
+     * @doc_url https://www.elastic.co/guide/en/elasticsearch/reference/current/run-as-privilege.html[Submit requests on behalf of other users]
```

Comment:
@delvedor, is this formatting correct for `@doc_url`? I see in the output how the URL renders, but I'm curious how the explanatory text in brackets actually displays.

>"docUrl": "https://www.elastic.co/guide/en/elasticsearch/reference/current/run-as-privilege.html[Submit requests on behalf of other users]",

Reply:
The `doc_url` should just be a plain URL, no markdown/asciidoc format :)